dev-docs/source/Testing/EngineeringDiffraction/EngineeringDiffractionTestGuide.rst

```diff
@@ -121,7 +121,7 @@ This test covers the Cropping functionality in the Calibration tab.
 
 7. Change the RB number to `Custom`.
 
-8. Set the ``Region Of Interest`` to ``Crop to Spectra`` and using ``Custom Spectra`` ``1200-1400`` (these spectrum numbers correspond to the South Bank).
+8. Set the ``Region Of Interest`` to ``Crop to Spectra`` and using ``Custom Spectra`` ``1200-2400`` (these spectrum numbers correspond to the South Bank).
    Please note that some custom spectra values may cause the algorithms to fail. Click ``Calibrate`` and a similar plot to before should appear but with only 2 subplots.
 
 9. Set the ``Region of Interest`` to ``Texture (20 spec)`` - there should be 20 spectra per run (5 tiled plot windows, 4 spectra per window).
```

Comment:
9. Set the ``Region of Interest`` to ``Texture (20 spec)`` **and  Click 'Calibrate'**?